### ISSUE TITLE
DateScale is inaccurate on scale of hours

### ISSUE THREAD

**Issue by joders** [bqplot/bqplot#967]
**Describe the bug**
Feeding time data into plots that is on the scale of less than a day this bug becomes visible. The plot shows the data displaced by the amount of the timezone offset used by the browser of the user. 

**To Reproduce**
```
import importlib
import bqplot
importlib.reload(bqplot)
from bqplot import *
xsc=DateScale()
ysc=LinearScale()
mark=Scatter(scales={'x':xsc,'y':ysc})
xax=Axis(scale=xsc)
yax=Axis(scale=ysc,orientation='vertical')
with mark.hold_sync():
    mark.x=np.array([
        "2019-01-01T00:00:00",
        "2019-01-02T00:00:00",
        "2019-01-03T00:00:00",
    ]).astype("datetime64[s]")
    mark.y=[0,1,0]
fig = Figure(marks=[mark],axes=(xax,yax))
fig
```

**Expected behavior**
I would expect the data points in the plot to be located at the dates that I'm feeding into the system.

**Screenshots**
![image](https://user-images.githubusercontent.com/12979604/68560175-d38d5e80-047a-11ea-94e8-3803aca4cd16.png)

**Environment:**
Reproduced this in Firefox and Chrome
`bqplot.__version__ == '0.12.0'`
Ubuntu 19.10

**Comment**
I already wrote a fix for the issue which I'm going to post in a few minutes/hours.

**Comment by SylvainCorlay** [comment]
@joders closing as I merged your PR - which should be in 0.12.8.

**Comment by joders** [comment]
nice. thanx
